pkg/web/static/js/usersManagement.js

```diff
@@ -29,6 +33,29 @@ document.addEventListener('htmx:afterSettle', (e) => {
       });
     };
   };
+
+  // Get user ID, if it exists, after the DOM settles.
+  const userID = document.getElementById('user-id');
+  const userDetailEP = `/v1/users/${userID?.value}?detail=password`;
+
+  if (e.detail.requestConfig.path === userDetailEP && e.detail.requestConfig.verb === 'get') {
+    const passwordInput = document.getElementById('password');
+    const confirmPassword = document.getElementById('confirm-password');
+    const changePasswordBtn = document.getElementById('change-password-btn');
+
+    // TODO: Convert to a reusable function for password matching.
+    const checkPasswords = () => {
```

Comment:
I tried to import this from `userProfile` but I think because the DOM changes, the IDs weren't recognized even though I imported those as well. I'll refactor this so that the function doesn't have to be copied.

Reply:
Sounds good - I'm assuming that if the element is looked up after the DOM refresh, it should find it; which means the element needs to be fetched inside the function call?

Reply:
Yes, in this case the element has to be fetched inside an `htmx:afterSettle` event or we'll get an `undefined` response.